shell/platform/windows/win32_flutter_window.h

```diff
@@ -0,0 +1,177 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_FLUTTER_WINDOW_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_FLUTTER_WINDOW_H_
+
+#include <windowsx.h>
+
+#include <string>
+#include <vector>
+
+#include "flutter/shell/platform/common/cpp/client_wrapper/include/flutter/plugin_registrar.h"
+#include "flutter/shell/platform/common/cpp/incoming_message_dispatcher.h"
+#include "flutter/shell/platform/embedder/embedder.h"
+#include "flutter/shell/platform/windows/angle_surface_manager.h"
+#include "flutter/shell/platform/windows/key_event_handler.h"
+#include "flutter/shell/platform/windows/keyboard_hook_handler.h"
+#include "flutter/shell/platform/windows/platform_handler.h"
+#include "flutter/shell/platform/windows/public/flutter_windows.h"
+#include "flutter/shell/platform/windows/text_input_plugin.h"
+#include "flutter/shell/platform/windows/win32_window.h"
+#include "flutter/shell/platform/windows/window_state.h"
+
+namespace flutter {
+
+// A win32 flutter window.  In the future, there will likely be a
+// CoreWindow-based FlutterWindow as well.  At the point may make sense to
+// dependency inject the native window rather than inherit.
+class Win32FlutterWindow : public Win32Window {
+ public:
+  Win32FlutterWindow();
+  Win32FlutterWindow(const char* title,
+                     const int x,
+                     const int y,
+                     const int width,
+                     const int height) noexcept;
+  ~Win32FlutterWindow();
+
+  // Run a Windows message pump that also pumps plugin messages.
+  void FlutterMessageLoop();
+
+  // Virtual method called by base Window when a resize occurs
```

Comment:
Don't re-comment declarations of overridden methods, except to say what class the override is from (see other engine code with `override` for examples to follow).

Reply:
Fixed in next commit